libs/common/byteutils.hpp

```diff
@@ -44,6 +41,58 @@ namespace iroha {
     return array;
   }
 
+  /**
+   * Convert string of raw bytes to printable hex string
+   * @param str
+   * @return
+   */
+  inline std::string bytestringToHexstring(const std::string &str) {
```

Comment:
What's the point?

Reply:
Main point is to allow users to pass their keys in hex format while binary strings are really uncomfortable to work with.
Btw, tests for converters exist but in develop branch: https://github.com/hyperledger/iroha/pull/714/files.

Reply:
Sorry for being vague, I mean what's the point of moving it out? It seems a bit out of that pr

Reply:
The problem was we need converters in Blob. Initially is contained only one of them and I decided to copy it from types.hpp - definitely not the best decision ever. Now I need one more converter so I decided to fix my old mistake and call function instead of copy its content. I really do not like their current place in types.hpp - it's really overloaded by functions of different purposes which makes almost all project parts dependent of types.hpp (you can do a simple test - change something there and almost all project will be rebuilt). So I decided to do a small refactor just here.